api/mod/src/routers/login.py

```diff
@@ -0,0 +1,26 @@
+"""Routes for login"""
+
+from fastapi import APIRouter, HTTPException
+
+from mod.src.models.login import LoginData
+
+ROUTER = APIRouter(
+    prefix="/login",
+    tags=["login"],
+)
+
+
+@ROUTER.post(
+    "/",
+    status_code=204,
+    responses={
+        401: {"description": "Failed to validate login"},
+    },
+    operation_id="login",
+)
+async def login(
+    login_data: LoginData,
+) -> None:
+    """Validate login via username and password"""
+    if login_data.username != "AnnotoUser#1337" or login_data.password != "test1234":
```

Comment:
I think it would be nice to have that in a method of `LoginData`, but it is just a placeholder here, so nils and I have to do that in our PR is guess ;)

Reply:
Great idea! But yeah, since this PR is mostly ready to be merged, let's keep it in mind for #81